iree/test/e2e/models/BUILD

```diff
@@ -25,7 +25,6 @@ iree_lit_test_suite(
     name = "lit",
     srcs = glob(
         ["*.mlir"],
```

Comment:
Nit: can you fold this all onto one line, now that it's got no excludes?

Reply:
Sure.